modules/microshift-install-rhel-image-mode-csi-vgloop.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-csi_{context}"]
+= Configuring container storage for the bootc image and the {microshift-short} CSI
+
+Create and allocate an additional LVM volume group loop device to be shared with the container and used by the {microshift-short} Logical Volume Manager (LVM) Container Storage Interface (CSI).
+
+.Prerequisites
+
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have SUDO permissions configured.
+
+.Procedure
+
+. Create a file to be used for LVM partitioning by running the following command:
++
+[source,terminal]
+----
+VGFILE=/var/lib/microshift-lvm-storage.img
+
+sudo losetup -f "${VGFILE}"
+----
+
+. Configure the partition as a loop device by running the following command:
++
+[source,terminal]
+----
+VGSIZE=1G
+
+sudo truncate --size="${VGSIZE}" "${VGFILE}"
+----
+
+. Query the loop device name by running the following command:
++
+[source,terminal]
+----
+$ VGLOOP=$(losetup -j ${VGFILE} | cut -d: -f1)
+----
+
+. Create a free volume group on the loop device by running the following command:
++
+[source,terminal]
+----
+$ sudo vgcreate -f -y rhel "${VGLOOP}"
+----
+//according to the MicroShift CSI driver requirements described in Storage Configuration
+//TODO are we missing steps?
```

Comment:
Not a step, maybe a reference to the CSI requirements?

Reply:
I placed a link before this module, just after the first storage module since we should know the CSI requirements by the time we get here.